contracts/azorius/Azorius.sol

```diff
@@ -81,14 +91,17 @@ contract Azorius is Module, IAzorius {
     error InvalidTxs();
     error InvalidArrayLengths();
 
+    /**
+     * Initial setup of the Azorius instance.
+     */
     function setUp(bytes memory initParams) public override initializer {
         (
-            address _owner,
+            address _owner,                 // TODO what should these say?
```

Comment:
@tbwebb22 what are these three setup parameters exactly? (`owner`, `avatar`, `target`)

Reply:
So `owner` is the address that can call all of the `onlyOwner` functions. 
The `target` is the contract that transactions are first passed to. 
And the `avatar` is the contract that the transactions will ultimately be executed by. 

In our case, these three will probably all always be the same - the Gnosis Safe. I guess a case when avatar and target could be different, is for some reason if transactions need to be passed through another module, or some other contract. But I'm not really sure of when or why that would be needed.